cypress/support/commands.js

```diff
@@ -29,13 +29,44 @@ import { addMatchImageSnapshotCommand } from 'cypress-image-snapshot/command';
 addMatchImageSnapshotCommand();
 
 Cypress.Commands.add('getByDataCy', (selector) => {
-  cy.get(`[data-cy="${selector}"]`);
+  cy.get(`[data-qa="${selector}"]`);
 });
 
 Cypress.Commands.add('register', (email = 'riot@qa.team', username = 'riot', password = '12345Qwert!') => {
   cy.request('POST', '/users', {
     email,
     username,
     password
+  }).then(response => {
+        cy.setCookie('drash_sess', response.body.user.token);
+      });
+    });;
```

Comment:
Extra semicolon at the end of the line. It's advisable to consistently follow the JavaScript standard style.